### PR TITLE
linux guest configuration and stap integration

### PR DESCRIPTION
**Pull request by doomedraven**

### CODE REVIEW

File: docs/book/src/installation/guest/linux.rst [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+Output should be something like **staprun:insert_module:x Module stap_ inserted from file path_to_stap_.ko**
+stap_.ko should be placed in /root/.cuckoo::
+
+    mkdir /root/.cuckoo
+    mv stap_.ko /root/.cuckoo/
+
+
+Disable firewall::
+    
+    sudo ufw disble
```
Comment (hsheep) [inline docs/book/src/installation/guest/linux.rst]: sudo ufw disble

File: docs/book/src/installation/guest/linux.rst [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    @reboot python path_to_agent.py
+
+Install dependencies::
+
+    sudo apt-get install systemtap gcc linux-headers-$(uname -r) 
+
+Compile Kernel extension::
+
+    wget https://raw.githubusercontent.com/cuckoosandbox/cuckoo/master/data/strace.stp
+    sudo stap -r $(uname -r) strace.stp -m stap_ -v
```
Comment (hsheep) [inline docs/book/src/installation/guest/linux.rst]: sudo stap -r $(uname -r) strace.stp -m stap_ -vvv

Comment (doomedraven) [inline docs/book/src/installation/guest/linux.rst]: you don't need -vvv

**Comment by jbremer** [comment]
@doomedraven Let me once you consider this PR ready to go.

**Comment by doomedraven** [comment]
@jbremer now is done :) , as strace.stp is only for x32/x64 arch, it not working on arm/mips probably some code adjudication required

but btw, for linux arm/mips/etc platforms on qemu you also should merge https://github.com/cuckoosandbox/cuckoo/pull/1239 as it fixes small paths checks

**Comment by jbremer** [comment]
Looks fine to me, thanks!

**Pull request merged**